api_tests/base/test_sloan_study.py

```diff
@@ -63,11 +62,9 @@ def providers(self, user):
     def flags(self, user):
         Flag.objects.filter(name__in=SLOAN_FLAGS, percent=50).update(everyone=None)
 
-    @override_settings(DEV_MODE=False)
```

Comment:
No longer using `DEV_MODE` to decide cookie status, a note in `website/settings/local.py` says `SECURE_MODE` is used to determine this in cookies,